deployment/CE05MOAS-G0917_Deploy.csv

```diff
@@ -0,0 +1,7 @@
+CUID_Deploy,deployedBy,CUID_Recover,recoveredBy,Reference Designator,deploymentNumber,versionNumber,startDateTime,stopDateTime,mooring.uid,node.uid,sensor.uid,lat,lon,orbit,deployment_depth,water_depth,notes,electrical.uid,assembly_template_revision
+EK20210730,S.Pearce,,,CE05MOAS-G0917-00-ENG000000,1,1,2021-07-30T16:45:00,,CGVCE-05MOAS-00000,CGVEH-GLDRCE-00917,CGCON-GLDRCE-00917,44.646,-124.128,,200,200,On rechargeable lithium batteries,,
+EK20210730,S.Pearce,,,CE05MOAS-G0917-01-PARADM000,1,1,2021-07-30T16:45:00,,CGVCE-05MOAS-00000,CGVEH-GLDRCE-00917,CGINS-PARADM-50336,44.646,-124.128,,200,200,On rechargeable lithium batteries,,
+EK20210730,S.Pearce,,,CE05MOAS-G0917-02-FLORTM000,1,1,2021-07-30T16:45:00,,CGVCE-05MOAS-00000,CGVEH-GLDRCE-00917,CGINS-FLORTM-06744,44.646,-124.128,,200,200,On rechargeable lithium batteries,,
+EK20210730,S.Pearce,,,CE05MOAS-G0917-03-ADCPAM000,1,1,2021-07-30T16:45:00,,CGVCE-05MOAS-00000,CGVEH-GLDRCE-00917,CGINS-ADCPAM-73246,44.646,-124.128,,200,200,On rechargeable lithium batteries,,
+EK20210730,S.Pearce,,,CE05MOAS-G0917-04-DOSTAM000,1,1,2021-07-30T16:45:00,,CGVCE-05MOAS-00000,CGVEH-GLDRCE-00917,CGINS-DOSTAM-00891,44.646,-124.128,,200,200,On rechargeable lithium batteries,,
+EK20210730,S.Pearce,,,CE05MOAS-G0917-05-CTDGVM000,1,1,2021-07-30T16:45:00,,CGVCE-05MOAS-00000,CGVEH-GLDRCE-00917,CGINS-CTDGVM-09692,44.646,-124.128,,200,200,On rechargeable lithium batteries,,

```

Comment:
These look ok, but no way to verify the sensor SNs - no FCP or sensor cal files in Vault.  Also checked RDB & Alfresco and didn't see any corresponding files.

Reply:
@rtravis-whoi I have now uploaded the calibration files to Alfresco (minus the ADCPAM docs since there is not a space for those in the usual calibration docs directories)

[Configuration](https://alfresco.oceanobservatories.org/alfresco/d/d/workspace/SpacesStore/685cd1eb-bd2c-419f-84f2-03b4c8956a37/8304-61003-917-00001_Configuration_Record.xlsx)

[PARAD](https://alfresco.oceanobservatories.org/alfresco/d/d/workspace/SpacesStore/4ca62f3b-04ae-4d3c-8443-59837f3ff432/PARAD-M_QSP2155_SN_50336_Calibration_2020-08-17.pdf)
[DOSTA](https://alfresco.oceanobservatories.org/alfresco/d/d/workspace/SpacesStore/516ade1a-7178-4c30-9594-2e21f89bcb48/DOSTA-M_Optode-4831_SN_891_Calibration_2020-06-14.pdf)
[FLORT](https://alfresco.oceanobservatories.org/alfresco/d/d/workspace/SpacesStore/52376cfe-49f5-4100-be35-7dcb0e048084/FLORT-M_FLBBCDSLC_SN_6744_Calibration_2021-03-18.pdf)
[CTDGV](https://alfresco.oceanobservatories.org/alfresco/d/d/workspace/SpacesStore/e569c30f-105f-428a-9634-2d549a73f8b5/CTDGV-M_SBE-Slocum_SN_9692_Calibration_2021-02-03.pdf)

Reply:
Yah - I just realized that there isn't an ADCPA folder in Alfresco....not really sure why.  ADCPA folder has been added.